### PR TITLE
Issue/645 delete confirmation

### PR DESCRIPTION
**Pull request by peeb**
- [x] Update release notes
- [ ] Test plan | Unit test | Integration test
- [ ] Copyright header
- [x] Code formatting
- [ ] Documentation

### CODE REVIEW

**Comment by peeb** [comment]
@gabemart I think the <select> dropdown needs some styling. Comments on that are especially welcome.

File: client/src/components/Users.jsx [hunk 1]
```diff
@@ -1,64 +1,55 @@
...
+function UserActionSelector({ user, onChange }) {
+  const { active, admin } = user;
+  return (
+    <select
+      className="UserActionSelector"
+      onChange={event => onChange(user, event.target.value)}
+      value="?"
+    >
+      <option value="?">...</option>
+      <option disabled key="user-edit" value="edit">
```
Comment (gabemart) [inline client/src/components/Users.jsx]: I don't think the 'key' props are strictly necessary here and below, but they don't do any harm

**Comment by gabemart** [comment]
Looks good to me

File: client/src/components/Users.jsx (hunk 1 above)
Comment (peeb) [inline client/src/components/Users.jsx]: I think I saw a console warning about flattened children (!) - but it's probably harmless.

**Pull request merged**